CHANGELOG.md

```diff
@@ -24,6 +24,11 @@ to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
  * `jj bisect run --command <cmd>` is deprecated in favor of
    `jj bisect run -- <cmd>`.
 
+ * `jj metaedit --update-committer-timestamp` is deprecated in favor of
```

Comment:
nit: Add ", which has the same behavior" or similar. As a user, I wouldn't be sure otherwise if there's some functional difference between the two flags. Actually, it might be better to also avoid "The former" below since that seems to suggest a functional difference. Maybe like this:

```md
* `jj metaedit --update-committer-timestamp` was renamed to
  `jj metaedit --force-rewrite` since the old name (and its help text)
  incorrectly suggests that the committer name and email would not be
  updated.
```